main.tf

```
@@ -84,7 +84,7 @@ data "template_file" "kubeadm-config-join" {
 
 resource "random_password" "certificate_key" {
   length           = 64
-  number           = false
+  numeric           = false
```

I think these changes need to be `terraform fmt`ed
```suggestion
  numeric          = false
```